terraform/workspace-variables/review_app_env.yml

```diff
@@ -3,7 +3,7 @@ RACK_ENV: production
 RAILS_ENV: deployed_development
 NODE_ENV: production
 RAILS_LOG_TO_STDOUT: true
-RAILS_MAX_THREADS: 5
+RAILS_MAX_THREADS: 20
```

Comment:
are we able to revert this before we merge?

Reply:
https://github.com/DFE-Digital/early-careers-framework/pull/1695#discussion_r800552898